zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/UserTaskTransformerTest.java

```diff
@@ -232,4 +238,33 @@ void shouldTransform(final String formId, final String parsedExpression) {
       }
     }
   }
+
+  @Nested
+  @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+  class ZeebePriorityTests {
+
+    Stream<Arguments> priority() {
+      return Stream.of(
+          Arguments.of(null, "50"),
+          Arguments.of("", "50"),
+          // Fails, default value with space not set Arguments.of(" ", "50"),
```

Comment:
Yeah there are ways to solve this, however in a more generic way of seeing this issue, since empty String ("") is being replaced by the parser with the default value, shouldn't the same be the case for Strings containing only whitespace characters?

Reply:
Yes, same as the `assignee` property

Reply:
@korthout Since there is a defined and documented default value for the Element Definition of _Priority_, but not limited to this field alone, I believe that the parser should be responsible to apply that default value. Setting the default value on the exported Record is also done but it cannot be considered default and be declared on the Element Definition, in my opinion, if it does not treat these 3 cases equally:
- Empty string ("")
- Null
- Whitespace string (" ")

At this moment the third case is not being handled correctly. As a solution for _Priority_ field specifically,  I am trimming the value of non-null Strings so as to be automatically set to the Default one.